pmlc/dialect/pxa/transforms/vectorize_mem.cc

```diff
@@ -320,6 +325,11 @@ struct VectorizeMemImpl {
     auto results = loopOp.results();
     for (auto res : results) {
       res.setType(nemMemrefType);
+      // Update also AffineYieldOp if type does not match
+      for (auto yieldOp : loopOp.getOps<AffineYieldOp>()) {
+        if (yieldOp.getOperand(0).getType() != nemMemrefType)
```

Comment:
Seems like the original code had a typo (I presume nemMemrefType is supposed to be newMemrefType). Probably should fix the name as part of this commit.